zyte_api/aio/client.py

```diff
@@ -148,3 +182,44 @@ async def _request(query):
                     session=session)
 
         return asyncio.as_completed([_request(query) for query in queries])
+
+    @staticmethod
+    def _build_extract_query(raw_query):
+        return {
+            _to_lower_camel_case(k): v
+            for k, v in raw_query.items()
+        }
+
+    async def extract(
+        self,
+        url: str,
+        *,
+        session: Optional[aiohttp.ClientSession] = None,
+        handle_retries: bool = True,
+        retrying: Optional[AsyncRetrying] = None,
+        **kwargs,
+    ) -> Awaitable[ExtractResult]:
```

Comment:
Current features, compared to `request_raw`:
- Pass top-level arguments instead of a dictionary.
- May pass `url` as a positional argument.
- Allows using Pythonic snake case for top-level query parameters.

Some potential features:
- Allow a `headers` parameter, which automatically fills the corresponding API parameters.
- Allow an `outputs` parameter, supporting a flag-like way of defining which outputs to enable.

The point of using `**kwargs` instead of mapping all parameters is forward-compatibility.